constant/instance_sharing.go

```diff
@@ -0,0 +1,6 @@
+package constant
```

Comment:
this folder is not following go folder structure convention.
consider placing this constant in instance sharing relevant types file (as done for other exported pkgs)

Reply:
@alexwo - I keep it open for now
@nirbenrey - this change will cause failures in sm-sap
I will update once it is done at the group chat.